eggdrop.conf

```diff
@@ -76,27 +75,19 @@ set offset "5"
 # If you're using virtual hosting (your machine has more than 1 IP), you
 # may want to specify the particular IP to bind to. You can specify either
 # by hostname or by IP. Note that this is not used for listening. Use the
-# 'listen-addr' variable to specify the listening address. If you are 
-# behind a NAT, you will likely want to leave this commented out as Eggdrop
-# should determine the correct IP on its own. 
+# 'listen' command to specify the listening address. If you are behind a
```

Comment:
"behind NAT" is better than "behing a NAT", no?

Reply:
It took me a while to figure out 'behing' was your typo, not mine :P

Reply:
I'm familiar with using this as "behind a NAT", and I think that is the vernacular for the term in this case.